src/detect-parse.c

```diff
@@ -2365,6 +2367,9 @@ void DetectParseFreeRegexes(void)
         if (r->study) {
             pcre_free_study(r->study);
         }
+        if (r->jit_stack) {
```

Comment:
is this still needed in this parsing work?

Reply:
Yes because these are helper functions and are called from code in `detect-pcre.c`